[2025-04-01 01:14:59,114] 	INFO 	14 	id_data_v2.py 	<module>() 	ID Data Started
[2025-04-01 01:29:41,169] 	INFO 	163 	id_data_v2.py 	<module>() 	ID Data Ended
[2025-04-01 01:29:41,948] 	INFO 	14 	id_data_v3.py 	<module>() 	ID Data Started
[2025-04-01 01:29:42,844] 	INFO 	40 	id_data_v3.py 	<module>() 	ID Data Ended
[2025-04-01 01:29:43,439] 	INFO 	19 	day_data.py 	<module>() 	Day Data Started
[2025-04-01 01:29:46,052] 	ERROR 	69 	day_data.py 	<module>() 	Data successfully appended to Google Sheets.
[2025-04-01 01:29:46,052] 	INFO 	80 	day_data.py 	<module>() 	Day Data Ended
[2025-04-01 01:29:46,052] 	INFO 	86 	day_data.py 	<module>() 	Day Data Fusion Started
[2025-04-01 01:29:48,586] 	ERROR 	145 	day_data.py 	<module>() 	Data successfully appended to Google Sheets.
[2025-04-01 01:29:48,586] 	INFO 	156 	day_data.py 	<module>() 	Day Data Fusion Ended
[2025-04-01 01:29:48,586] 	INFO 	163 	day_data.py 	<module>() 	Day Data Combined Started
[2025-04-01 01:29:51,744] 	ERROR 	196 	day_data.py 	<module>() 	Data successfully added to Google Sheets.
[2025-04-01 01:29:51,744] 	INFO 	207 	day_data.py 	<module>() 	Day Data Combined Ended
[2025-04-01 01:29:52,599] 	INFO 	31 	pair_data.py 	<module>() 	Pair Data Started
[2025-04-01 01:31:08,716] 	ERROR 	111 	pair_data.py 	<module>() 	Error occurred: Out of range float values are not JSON compliant
[2025-04-01 01:31:08,716] 	ERROR 	113 	pair_data.py 	<module>() 	Retrying in 30 seconds... (Attempt 2/3)
[2025-04-01 01:31:38,855] 	ERROR 	111 	pair_data.py 	<module>() 	Error occurred: Out of range float values are not JSON compliant
[2025-04-01 01:31:38,855] 	ERROR 	113 	pair_data.py 	<module>() 	Retrying in 30 seconds... (Attempt 3/3)
[2025-04-01 01:32:09,030] 	ERROR 	111 	pair_data.py 	<module>() 	Error occurred: Out of range float values are not JSON compliant
[2025-04-01 01:32:09,030] 	ERROR 	116 	pair_data.py 	<module>() 	All retries failed.
[2025-04-01 01:32:09,030] 	ERROR 	121 	pair_data.py 	<module>() 	Error occurred during Pair Data process. Error: Out of range float values are not JSON compliant
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/requests/models.py", line 510, in prepare_body
    body = complexjson.dumps(json, allow_nan=False)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
ValueError: Out of range float values are not JSON compliant

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "pair_data.py", line 107, in <module>
    gs.values_append("Master", {"valueInputOption": "USER_ENTERED"}, {"values": df_values})
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/gspread/spreadsheet.py", line 150, in values_append
    r = self.client.request("post", url, params=params, json=body)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/gspread/client.py", line 79, in request
    response = getattr(self.session, method)(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/google/auth/transport/requests.py", line 537, in request
    response = super(AuthorizedSession, self).request(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/requests/sessions.py", line 575, in request
    prep = self.prepare_request(req)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/requests/sessions.py", line 484, in prepare_request
    p.prepare(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/requests/models.py", line 370, in prepare
    self.prepare_body(data, files, json)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/requests/models.py", line 512, in prepare_body
    raise InvalidJSONError(ve, request=self)
requests.exceptions.InvalidJSONError: Out of range float values are not JSON compliant
[2025-04-01 01:32:09,031] 	INFO 	136 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2025-04-01 01:32:09,553] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: auniBTC-FBTC, Address: 0xe2BB11d6b6A39e55762F5e14D632F0981198b3A7, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:10,668] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: ajEUR-USDC, Address: 0x85B0533848E2a97069e2a5937fa927cA9133AF97, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:10,879] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: ajCHF-USDC, Address: 0x786C5B03C7A0a88A4cE6860a0D4aB2a1a0864e57, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:11,099] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aenzoBTC-ynBTCk, Address: 0x87BB1366C3d7AFc80bB0cEc7800FcE29D11AdBd7, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:11,306] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aankrBNB-XVS, Address: 0x0748fb34bcb68d2c055cDfCb9F5bcB3549FcB456, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:11,917] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aXCAD-WBNB, Address: 0xe2624920C88b470B78EFAdfdE51Dd0E2C841934B, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:12,075] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aXCAD-BUSD, Address: 0x0Cd7F0d5478FEFe59F2B9F5aBa623706f2924e77, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:12,536] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWUSDR-USDC, Address: 0x3f8062cf591a918Bc3B183fa241ec49efBd30e2b, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:12,711] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWUSDR-DOLA, Address: 0xdB6BfF81311C5C4F72668D9C4c8a5E2CB4584A82, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:14,015] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-STKBNB, Address: 0x84b78452A97C5afDa1400943333F691448069A29, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:14,546] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-RBNB, Address: 0x3cAd1CBA60f695Ec7FA522Ec486f55D77C270b18, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:15,049] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-LPBNB, Address: 0x47600Bc3aE9b5b97ef92A55E550066944fE17670, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:15,401] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-ERN, Address: 0xc609daA86FF43F34Eef52C468CbC4934eF8D2502, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:15,551] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-EDU, Address: 0x4bD39cfb05545c4238316dA69eCadA1b4CaABC05, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:16,137] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aWBNB-BIFI, Address: 0xCC0b8D28e7c31555a3889CDc05AF55bfF57c368A, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:16,630] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-USDV, Address: 0x739e561786C920D025d57eD99be5D4EcA3458E3E, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:17,276] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-PLAY, Address: 0xa7aBA8136C5758B47312233C684547B651FCF775, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:18,299] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDT-ERN, Address: 0x8DCDF3CbD3a588a2e2FB607BdC9180e4FefA583b, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:18,879] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aUSDC-WUSDR, Address: 0xcbdE6C4f202eCc13267fD53913bc44B4Dd6eF998, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:19,586] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aSTKBNB-THE, Address: 0xF1b7007694DBcba004DB73473CCa3c5aDd3803E4, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:19,752] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aSOLV-WBNB, Address: 0xBB59e13bD9B6Bf6f3E0421939649Fd47927d2206, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:20,658] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aORBS-WBNB, Address: 0x130348553b3dEA5d65767Dc390eFF257F9a9181d, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:20,829] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aOATH-ERN, Address: 0xDF5751BBEf0241878852253F8541A02c3Cbc2dd1, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:21,679] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aJBRL-USDC, Address: 0xd63bBA6078a328b93758Cc7556e21D3380397D41, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:21,851] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aID-WBNB, Address: 0xB66566fD115CAf17633504A5165a319F17D9F096, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:22,538] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aHAY-FRAX, Address: 0x30F8542ede13c68c5B4ed2922401c8BA63d88e59, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:22,748] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aGRAIN-WBNB, Address: 0x1f060bE9D1D3Bd3ACC5392aC066C6B437780ABe0, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:23,815] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aFRAX-ERN, Address: 0x1C2D4F310aeA872C5BD55790D2D78Ca7B31992a3, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:24,448] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-inETH, Address: 0x52e3e02B5961292C388A1faE1ce4EaD724D4E806, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:25,213] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-STKBNB, Address: 0x8b41d72def3867E19f2285B45DA655e24928018F, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:25,863] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aETH-ANKRETH, Address: 0xC9dF4aE113fA7CB67CE1779cB1560D1f43c2D373, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:27,369] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aCASH-USDT, Address: 0x45F1Fdb00fB78174196Fa109A27Cdbd07f67F85e, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:28,376] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-THE, Address: 0x1335Ffc90516b022425dBcFe4eA3610aA45f6C3E, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:28,754] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-ETH, Address: 0x6B32E9178a6670dc9b19a74FE5080C54ea5347C7, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:28,886] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aBNBX-ERN, Address: 0xc7426Dce489E6b2d580Fd480Ef72D57e5d7643f5, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:29,456] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: aARKEN-WBNB, Address: 0xCB995DaE5238c1e08C7289b335c80ED8d129A46b, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:31,383] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-8-THE a082, Address: 0x8929E532E551c82f5736aa18C78440691BC542D9, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:31,954] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-47-THE af58, Address: 0xDcc442489f861275B327D319534187592E193b36, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:32,929] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-36-THE Bc6a, Address: 0xFd60A2B164C86751Df65c8cf895F7b07e5A48C35, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:33,890] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-15-THE d198, Address: 0x7cEfCEa15158Df116534C75F2f677d4D1f077028, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:34,253] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: IV-11-THE 2ff5, Address: 0xe1Ad12B9f713710a8FFF593eB1c6E5F82d3C0b3E, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:34,388] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: DEShare E25c, Address: 0x52ddB7BD7CA73a20b6334cC08fCACa9906992a15, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:34,573] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: DEShare A2bB, Address: 0x9545d5399E50a8d9B9bE239BD7117b5655a9A45D, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:34,720] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: DEShare 638a, Address: 0xaB3790c5FaB8AFf0A41640Eb2c7b280289c657E5, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:35,040] 	ERROR 	176 	pair_data.py 	<module>() 	Error occurred during Pair Data Fusion process. Pair: DEShare 31eD, Address: 0x37CbdBb97C34ba577ADbFbB80b517cD6caB636E8, Error: "None of [Index(['id', 'date', 'tvlUSD', 'volumeUSD', 'volumeToken0', 'volumeToken1',\n       'token0Price', 'token1Price', 'feesUSD', '__typename'],\n      dtype='object')] are in the [columns]"
[2025-04-01 01:32:44,672] 	ERROR 	215 	pair_data.py 	<module>() 	Data successfully appended to Google Sheets.
[2025-04-01 01:32:44,672] 	INFO 	226 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2025-04-01 01:32:44,672] 	INFO 	233 	pair_data.py 	<module>() 	Pair Data Combined Started
[2025-04-01 01:33:46,766] 	ERROR 	279 	pair_data.py 	<module>() 	Data successfully added to Google Sheets.
[2025-04-01 01:33:46,766] 	INFO 	290 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2025-04-01 01:33:47,703] 	INFO 	31 	bribe_data.py 	<module>() 	Bribe Data Started
[2025-04-01 01:33:52,481] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aankrBNB-XVS 33f4: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:33:56,842] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDC/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:33:59,666] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-GHNY/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:06,483] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aEQB-WBNB 982b: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:12,718] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBTCB-ynBTCk CC09: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:20,202] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:20,210] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.meowrpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:20,217] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed.bnbchain.org for nan: Address nan must be provided as a string
[2025-04-01 01:34:20,225] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.defibit.io for nan: Address nan must be provided as a string
[2025-04-01 01:34:20,232] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.ninicoin.io for nan: Address nan must be provided as a string
[2025-04-01 01:34:20,239] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.rpc.blxrbdn.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:23,314] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-MAI/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:27,888] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:27,897] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.meowrpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:27,903] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed.bnbchain.org for nan: Address nan must be provided as a string
[2025-04-01 01:34:27,910] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.defibit.io for nan: Address nan must be provided as a string
[2025-04-01 01:34:27,917] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.ninicoin.io for nan: Address nan must be provided as a string
[2025-04-01 01:34:27,924] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.rpc.blxrbdn.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:28,021] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-IDIA/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:30,892] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BADAI/WBNB 1E90: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:33,793] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:33,800] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.meowrpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:33,808] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed.bnbchain.org for nan: Address nan must be provided as a string
[2025-04-01 01:34:33,814] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.defibit.io for nan: Address nan must be provided as a string
[2025-04-01 01:34:33,821] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.ninicoin.io for nan: Address nan must be provided as a string
[2025-04-01 01:34:33,828] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.rpc.blxrbdn.com for nan: Address nan must be provided as a string
[2025-04-01 01:34:35,388] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aSINGULARRY-WBNB 9A37: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:38,830] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-PION/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:41,921] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-XCAD 7Fcf: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:48,366] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aenzoBTC-ynBTCk Af96: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:52,158] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HAY/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:34:55,623] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-MAI/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:05,507] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for auniBTC-BTCB 5E0a: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:11,566] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-DOLA/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:22,371] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-TON 8ad0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:25,276] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-DJED/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:27,912] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-GRAIN/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:30,533] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HBR/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:36,300] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/DEUS: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:40,863] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWoD-WBNB 00B1: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:44,864] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-ETH 2e3d: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:53,678] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-Bridged mwBETH/wBETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:35:57,888] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for ajEUR-USDC b314: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:00,338] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-XCAD/BUSD OLD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:02,831] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-RBNB 5772: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:08,553] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BTCB/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:13,625] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-ERN f047: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:24,127] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-TAROT/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:28,237] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aTWT-ANKRBNB B039: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:33,794] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-PENDLE 76c9: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:42,792] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BUSD/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:46,952] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/ANKR: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:50,977] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-CGPT/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:36:57,768] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-AST 3459: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:04,516] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BNBx/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:14,235] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-DAO/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:20,906] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BTCB/XCAD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:31,474] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:36,322] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-cpTHE/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:40,318] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-HAY/CUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:47,780] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for auniBTC-FBTC e40b: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:37:54,647] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-TWT/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:02,663] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/CHAM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:09,847] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-WBNB/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:18,055] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-DAO/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:23,964] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-FRAX/CUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:34,331] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aLTC-WBNB 531d: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:37,704] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BNBx/ETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:41,310] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-PRIMAL/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:47,304] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MAI/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:38:55,101] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-frxETH/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:07,114] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-THE DCDc: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:13,515] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aXCAD-BUSD 3613: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:24,407] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-XCAD/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:27,463] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-DEUS eCF4: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:30,463] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare 9287: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:33,413] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-uniBTC/brBTC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:40,371] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-OATH/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:46,529] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FEAR/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:55,317] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-AICELL 0Dbb: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:39:59,796] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-DOLA/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:03,619] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-XCAD/WBNB OLD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:11,026] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare a373: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:15,450] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BTCB/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:19,576] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-RDNT 97e8: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:28,430] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/XCAD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:34,276] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-38-THE E62E: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:41,748] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-30-THE e5c9: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:48,128] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FEAR/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:40:58,774] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-MORE/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:04,773] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-ANKR eE80: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:15,438] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDV/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:20,669] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aSFRXETH-FRXETH 2C3A: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:34,616] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-jBRL/BRZ: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:39,358] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDC/liveThe: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:43,597] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BTCB/GHNY: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:52,949] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HAY/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:55,786] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-37-THE F310: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:41:58,609] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-XCAD/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:01,115] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-RDNT 6f46: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:11,879] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aDAO-ankrBNB 0fB0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:17,759] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aTWT-WBNB 5196: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:25,851] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aID-ANKRBNB 32c6: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:29,216] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-COCO effD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:32,432] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-cpTHE/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:35,237] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-AVAX/CHAM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:46,066] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/rBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:49,045] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-USDT 3e64: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:51,847] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-sfrxETH/frxETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:42:55,368] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-XCAD/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:02,878] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDT/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:06,630] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-UNW/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:11,149] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-29-THE f060: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:23,289] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-IPAD/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:27,097] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/PLAY: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:31,432] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANDY-WBNB B254: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:37,204] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-39-THE EB04: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:41,949] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/multiBTC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:46,282] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:43:46,289] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.meowrpc.com for nan: Address nan must be provided as a string
[2025-04-01 01:43:46,296] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed.bnbchain.org for nan: Address nan must be provided as a string
[2025-04-01 01:43:46,304] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.defibit.io for nan: Address nan must be provided as a string
[2025-04-01 01:43:46,311] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc-dataseed1.ninicoin.io for nan: Address nan must be provided as a string
[2025-04-01 01:43:46,318] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://bsc.rpc.blxrbdn.com for nan: Address nan must be provided as a string
[2025-04-01 01:43:46,377] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBTCB-STKBNB 51b5: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:54,684] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-HAY/USD+: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:43:58,398] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-42-THE d5bb: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:03,068] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:13,095] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-LIBERA/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:15,771] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBTCB-WBNB 4D3F: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:18,331] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for asolvBTC-BTCB 0995: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:21,309] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-12-THE 2367: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:26,362] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BNBx/SD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:29,352] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-7-THE deb0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:31,914] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aAITECH-WBNB 27B6: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:34,502] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDC/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:45,692] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-cpTHENA/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:48,746] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-THE b15b: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:51,315] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-KOMA 78E4: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:44:54,969] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-DAI/MAI: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:04,587] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-14-THE B802: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:08,043] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aHAY-FRXETH 275E: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:11,997] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HAY/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:19,966] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDC/FTM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:24,177] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWEETH-ETH 0D39: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:28,894] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-XVS eE7D: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:39,146] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFIL-WBNB 0836: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:42,708] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDV/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:46,207] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aHAY-WBNB 0252: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:54,290] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDT+/USD+: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:45:59,450] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-BLOCK 5DD2: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:03,420] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-CHAM/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:19,981] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-FLOKI FC3e: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:24,490] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aGRAIN-WBNB 6BC7: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:28,288] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-BTC 5f88: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:39,060] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/MATIC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:41,679] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/MGP: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:44,338] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aSOLV-WBNB 62fb: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:46,924] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-ETH/frxETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:54,346] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MATIC/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:46:58,870] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HAY/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:03,132] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MIX/MATIC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:13,333] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/LIQ: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:16,466] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HBR/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:18,913] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aLISUSD-CHEEMS E491: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:22,126] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MPX/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:30,607] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDC/MIX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:36,633] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-sfrxETH/FXS: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:48,186] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/HBR: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:51,241] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-5-THE 740b: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:53,791] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-36-THE Bc6a: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:47:56,302] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-C98/CUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:02,961] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-9-THE 3fd0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:06,189] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/frxETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:10,412] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-ankrBNB/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:15,466] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-CaptainBNB/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:20,194] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-DAO/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:24,375] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDC/MGP: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:32,284] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-BTCB/Bridged mBTC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:36,889] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/FITFI: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:40,297] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-47-THE af58: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:47,374] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-DOLA/CUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:50,539] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-LPBNB Ba2b: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:53,138] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/JARVIS: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:48:56,142] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-wstETH bEcD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:02,504] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MULTI/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:05,131] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-OVN/USDT+: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:07,997] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/MGP: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:10,472] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-FDUSD C4F3: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:16,256] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-FRAX/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:20,911] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-DUSD/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:31,246] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-eTHENA/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:35,635] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-27-THE fEa0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:40,118] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-19-THE 6683: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:47,367] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aID-WBNB E753: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:49,975] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/BIFI: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:52,454] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-ABOND/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:49:54,937] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-32-THE 3357: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:01,944] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MAI/FXS: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:05,019] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-45-THE 1B9C: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:08,667] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aGRAIN-WBNB EAD0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:11,197] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-33-THE 025d: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:17,260] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-fBOMB/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:20,262] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BNBx/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:23,870] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDC/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:26,577] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-LQDR 0D12: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:37,379] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-THE 1aCB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:39,746] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-EDU 6feA: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:42,269] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-WBNB/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:44,432] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/SLM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:51,816] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/CHAM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:54,470] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/ankrETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:57,328] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BUSD/SLM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:50:59,795] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:04,841] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/FLOKI: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:09,857] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MAI/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:23,119] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MULTI/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:27,020] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FIL/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:30,417] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-AVAX/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:36,181] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-dXCAD/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:39,479] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/stkBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:42,211] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-wUSDR/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:44,962] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETS Alpha/USD+: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:54,122] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDC/dXCAD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:56,789] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/PRIMAL: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:51:59,254] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/LINK: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:03,022] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FS/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:08,710] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDT/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:12,116] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-XCAD/THE OLD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:14,875] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-BUSD/SLM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:17,841] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:23,752] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/LIQ: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:26,527] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare a3E3: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:29,094] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-SD/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:31,653] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/TONCOIN: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:35,882] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-MATIC 1E3F: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:38,508] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FS/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:41,079] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-FRAX/sFRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:44,481] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-STKBNB 668C: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:52,622] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/TAROT 8A8b: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:55,238] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-wUSDR/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:57,726] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDT/USDC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:52:59,806] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-MIX/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:05,280] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FEAR/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:08,502] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-USDC fBF2: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:10,967] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-WBNB 8e3A: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:14,257] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aSnBNB-WBNB 6708: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:21,066] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-PLAY B74d: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:24,163] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aankrBNB-DUSD 87b1: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:27,267] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-TON 80F2: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:30,108] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-ERN 67D3: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:35,601] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-LQDR/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:40,737] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USD+/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:44,632] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-IDIA/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:52,152] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:55,100] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-DAO/BUSD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:53:57,862] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-WBNB/LQDR: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:00,602] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-43-THE 8423: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:10,836] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/FEAR: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:13,645] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-40-THE CC08: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:17,211] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/KOL: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:22,781] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-FTM/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:25,380] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-IDIA/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:27,846] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-qTHE/THE 2ffA: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:30,660] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-BNBx/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:36,226] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-LIBERA/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:39,056] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-WBNB/BIFI: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:41,682] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-HAY/ANKR: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:44,682] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-HAY/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:51,780] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-BTCB/multiBTC: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:55,041] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-cpTHENA/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:54:57,574] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-ETH/PLAY: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:00,693] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-PLAY/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:08,251] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-HAY/FRAX: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:12,120] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/TAROT 7086: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:15,125] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/IPAD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:21,913] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDV/USDT: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:24,689] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-ETH/pxETH: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:27,664] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-xZOOMER/THE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:30,307] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-USDT/TTM: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:36,203] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for sAMM-USDT/USDa: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:38,851] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-axlLqdr/WBNB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:41,283] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-fBOMB/sliveThe: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:44,112] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for vAMM-PREON/STAR: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:50,309] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-35-THE 5736: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:53,890] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-41-THE CBDe: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:55:58,318] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-WBNB 9555: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:06,090] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aCGPT-BUSD 68C0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:09,453] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFTM-WBNB 4Fee: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:13,194] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aCGPT-BUSD 4536: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:19,518] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWUSDR-USDC aa92: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:22,273] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-FLOKI EF58: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:24,890] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-LQDR 3DCD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:27,417] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aDOGE-WBNB F0cF: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:34,330] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aDOGE-WBNB 3427: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:39,594] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-USDT 6071: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:43,347] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-EDU da6F: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:51,252] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-FRAX 6e92: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:55,077] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-ANKRETH 33e9: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:56:58,724] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aDAO-BUSD d6ea: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:04,613] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-ANKR 0808: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:07,145] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-OATH d561: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:09,723] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aLTC-WBNB 0C62: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:12,315] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aHAY-FRXETH 0A4F: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:20,585] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aXCAD-BUSD E405: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:23,778] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-USDT 3aBD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:27,270] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-FRXETH A888: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:34,650] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFRXETH-FRAX A59c: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:47,785] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-ANKRETH 63bd: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:57:54,940] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBNBX-THE 1A05: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:04,033] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWUSDR-DOLA 0876: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:07,615] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aSTKBNB-THE 27D7: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:10,223] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-BTE 5bc0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:12,870] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-LINK 587A: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:19,586] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-BTCB 7be8: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:25,454] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aANKRBNB-THE 4a5f: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:34,127] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFRAX-ERN 18ab: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:40,244] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aERN-OATH 4A7D: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:46,274] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBTCB-STKBNB F31F: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:49,531] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aLVL-WBNB E6EE: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:58:53,783] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-STKBNB 7ea9: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:05,611] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aARKEN-WBNB 44D0: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:12,400] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-ERN 578A: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:23,045] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aAXLLQDR-WBNB e1ce: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:27,704] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-STKBNB 0537: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:34,112] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFRXETH-WBNB e7a9: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:38,207] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-OOE 1974: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:41,811] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aCSIX-USDT 9a14: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:47,654] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFRXETH-RDNT a427: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:51,332] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aOATH-ERN 75df: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:54,348] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aAXLLQDR-WBNB Aa30: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 01:59:57,291] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aPNP-WBNB 07c3: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:06,273] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aAITECH-WBNB E8d7: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:12,247] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aFRXETH-WBNB 9cfd: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:28,834] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-DUSD 88ab: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:31,687] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aOATH-ERN 2F85: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:35,021] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aankrBNB-XVS 2206: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:42,950] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aSOL-WBNB eBFD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:46,090] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aUSDT-LISTA ff22: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:49,712] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aTOKEN-WBNB d726: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:00:56,869] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aynBNB-WBNB 32Fc: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:00,053] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-BTCB fc90: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:03,093] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-USDV 9FEf: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:06,137] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aETH-inwbETH Be13: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:13,825] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aBKN-USDT 013D: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:16,526] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-VINU 00d7: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:19,058] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aZRO-WBNB 98Eb: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:21,615] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-RWA 4aAD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:27,482] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aMONKY-WBNB Cd00: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:31,259] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for aWBNB-SDM 152E: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:35,240] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-8-THE a082: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:45,230] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-23-THE c02E: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:47,980] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-44-THE Bc06: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:51,650] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare 6055: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:54,187] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for IV-26-THE 2c1f: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:01:59,852] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare f117: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:02,600] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare A2bB: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:05,303] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare 3E21: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:07,968] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare 31eD: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:16,740] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare a32e: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:21,203] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare 733f: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:26,113] 	ERROR 	78 	bribe_data.py 	<module>() 	Error occurred while fetching bribes from https://binance.llamarpc.com for DEShare 8996: 429 Client Error: Too Many Requests for url: https://binance.llamarpc.com/
[2025-04-01 02:02:36,909] 	ERROR 	137 	bribe_data.py 	<module>() 	Data successfully appended to Google Sheets.
[2025-04-01 02:02:36,909] 	INFO 	148 	bribe_data.py 	<module>() 	Bribe Data Ended
[2025-04-01 02:02:37,531] 	INFO 	17 	revenue_data.py 	<module>() 	Revenue Data Started
[2025-04-01 02:03:04,594] 	ERROR 	81 	revenue_data.py 	<module>() 	Data successfully added to Google Sheets.
[2025-04-01 02:03:04,594] 	INFO 	92 	revenue_data.py 	<module>() 	Revenue Data Ended
[2025-04-01 02:03:05,188] 	INFO 	16 	revenue_data_v2.py 	<module>() 	Revenue Data Started
[2025-04-01 02:03:18,750] 	ERROR 	83 	revenue_data_v2.py 	<module>() 	Data successfully added to Google Sheets.
[2025-04-01 02:03:18,750] 	INFO 	94 	revenue_data_v2.py 	<module>() 	Revenue Data Ended
[2025-04-01 02:03:19,325] 	INFO 	29 	tvl_data.py 	<module>() 	TVL Data Started
[2025-04-01 02:05:36,035] 	INFO 	91 	tvl_data.py 	<module>() 	TVL Data Ended
[2025-04-01 02:05:36,035] 	INFO 	106 	tvl_data.py 	<module>() 	TVL Data Fusion Started
[2025-04-01 02:06:33,462] 	INFO 	178 	tvl_data.py 	<module>() 	TVL Data Fusion Ended
[2025-04-01 02:06:33,462] 	INFO 	185 	tvl_data.py 	<module>() 	TVL Data Combined Started
[2025-04-01 02:06:44,905] 	ERROR 	233 	tvl_data.py 	<module>() 	Data successfully appended to Google Sheets.
[2025-04-01 02:06:44,905] 	INFO 	244 	tvl_data.py 	<module>() 	TVL Data Combined Ended
[2025-04-01 02:06:45,783] 	INFO 	27 	fee_tvl_data.py 	<module>() 	Fee TVL Data Started
[2025-04-01 02:06:54,657] 	ERROR 	66 	fee_tvl_data.py 	<module>() 	Data successfully appended to Google Sheets.
[2025-04-01 02:06:54,658] 	INFO 	77 	fee_tvl_data.py 	<module>() 	Fee TVL Data Ended
